### PR TITLE
feat(Sprint-details): Resolved Current sprint button.

### PR DESCRIPTION
**Pull request by AvinashMahanthi**
Signed-off-by: AvinashMahanthi <avinashfab5678@gmail.com>

### Functionality:
Show Current sprint button when Sprint is in Under progress

### Solution:
Added dropdown for Sprint status, hiding the button when Status is Completed.

### Risk level:
- [ ] high 
- [X] medium
- [ ] low

### How to test:

- ng serve
- firebase emulators:start
- Go to board, check the sprint details.

### Screenshot:
image
Screenshot (218)

### CODE REVIEW

File: src/app/body/board/board.component.html [hunk 1]
```diff
@@ -17,7 +17,8 @@ <h3 class="fontHeading col-md-8 col-6" style="font-size: 24px;">Board</h3>
                         <app-feature-card [cardName]="'Business'" [tasks]="item.TotalBusinessTask"></app-feature-card>
                     </div>
                     <div class="col-md-3 col-12" type="button" (click)="showTasks('Development')">
-                        <app-feature-card [cardName]="'Development'" [tasks]="item.TotalDevelopmentTask"></app-feature-card>
+                        <app-feature-card [cardName]="'Development'" [tasks]="item.TotalDevelopmentTask">
+                        </app-feature-card>
```
Comment (shivamgoyal7) [inline src/app/body/board/board.component.html:21]: undo this change

File: src/app/body/board/board.component.html [hunk 2]
```diff
@@ -27,7 +28,8 @@ <h3 class="fontHeading col-md-8 col-6" style="font-size: 24px;">Board</h3>
                     </div>
                 </div>
 
-                <app-sprint-details [currentSprintName]="currentSprintName" [StartDate]="item.StartDate" [EndDate]="item.EndDate" [Status]="item.Status" (changeSprint)="changeSprintName($event)"></app-sprint-details>
+                <app-sprint-details [currentSprintName]="currentSprintName" [StartDate]="item.StartDate" [EndDate]="item.EndDate" [Status]="item.Status" (changeSprint)="changeSprintName($event)">
+                </app-sprint-details>
```
Comment (shivamgoyal7) [inline src/app/body/board/board.component.html:32]: undo this change

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]
Add screenshot of UI

File: src/app/body/board/board.component.html (hunk 2 above)
Comment (AvinashMahanthi) [reply src/app/body/board/board.component.html]: Done

File: src/app/body/board/sprint-details/sprint-details.component.ts [hunk 3]
```diff
@@ -31,13 +32,13 @@ export class SprintDetailsComponent implements OnInit {
...
-    
-    this.sprintStatus = "Completed";
+  async updateSprintStatus(updateStatus: string ) {
+    this.sprintStatus = updateStatus;
+    this.enableLoader = true;
     const callable = this.functions.httpsCallable('updateSprintStatus');
 
     try {
-      const result = await callable({ CurrentSprintName: this.currentSprintName, SprintStatus: this.sprintStatus }).toPromise();
+      const result = await callable({ CurrentSprintName: this.currentSprintName, SprintStatus: updateStatus }).toPromise();
```
Comment (shivamgoyal7) [inline src/app/body/board/sprint-details/sprint-details.component.ts:41]: Here in line 41 this.sprintstaus will come instead of updatestatus

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]

File: src/app/body/board/sprint-details/sprint-details.component.ts (hunk 3 above)
Comment (shivamgoyal7) [reply src/app/body/board/sprint-details/sprint-details.component.ts:41]: If u are using update status then remove line36

**Review by shivamgoyal7 — APPROVED** [review]
ship it

**Review by viv9k — CHANGES_REQUESTED** [review]
Fix the required changes

**Comment by shivamgoyal7** [comment]
resolve conflict

**Comment by viv9k** [comment]
Why is this PR closed, Please reopen it

**Comment by AvinashMahanthi** [comment]
Due to the latest changes some files have been corrupted in this branch, I had done this on the other branch ie. PR No: #130

**Pull request closed without merging**